bowling/inc/lane.hpp

```diff
@@ -1,8 +1,24 @@
 #pragma once
 
+#include "singleGame.hpp"
+
+#include <iostream>
+#include <string>
+#include <vector>
+
 class Lane {
 private:
+    std::vector<SingleGame> gamesInLine_{};
+    std::string laneName_{};
+
```

Comment:
Nice to have: create private member:
`std::string fileName_`
that would be initialized in constructor.
parseFile() and parseLaneName() could use this member instead of function arguments.